Rakefile

```diff
@@ -84,6 +86,63 @@ task :update_documentation do
   File.open("BuildingSync.xsd", 'w') { |f| f << doc.to_xml(:indent => 2) }
 end
 
+desc 'Get report of naming collisions between schemas'
+task :check_collisions do
```

Comment:
@macintoshpie -- can you have this run within an rspec. Probably create a new file called check_collisions_spec.rb. I may be missing something here though.